### PR TITLE
IGNORE - testing pr checks failed dialog on latest beta

### PR DESCRIPTION
**Pull request by tidy-dev**

### CODE REVIEW

File: app/test/unit/git/tag-test.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('git/tag', () => {
 
       const commit = await getCommit(repository, 'HEAD')
       expect(commit).not.toBeNull()
-      expect(commit!.tags).toEqual(['my-new-tag'])
+      expect(commit!.tags).toEqual(['my-nw-tag'])
```
Comment (tidy-dev) [inline app/test/unit/git/tag-test.ts]:
```suggestion
      expect(commit!.tags).toEqual(
      ['my-nw-tag']
      )
```

**Review by tidy-dev — COMMENTED** [review]
Attaching a file to see what our markdown renderer does with it

[README.md](https://github.com/user-attachments/files/15277635/README.md)

**Review by LLotme — COMMENTED** [review]
..

**Review by LLotme — COMMENTED** [review]
.

**Review by LLotme — COMMENTED** [review]
.

**Review by LLotme — COMMENTED** [review]
ok

**Pull request closed without merging**